.storybook/main.js

```diff
@@ -0,0 +1,51 @@
+const path = require("path");
+const { storyTree } = require("./storyTree");
+
+const stories = ["./welcome.story.js"];
+storyTree.forEach(catList => {
+  catList.components.forEach(component => {
+    const path = `../packages/${component}/stories/${component}`;
+    stories.push(
+      `${path}.stories.mdx`,
+      `${path}.stories.js`,
+      `${path}.examples.stories.js`,
+      `${path}.backyard.stories.js`,
+      `${path}.tests.stories.js`
+    );
+  });
+});
+stories.push("./components/CodeViewer/CodeViewer.story.js");
+stories.push("../packages/**/*.stories.js");
```

Comment:
I've refactored all existing component stories to utilize the `storyTree`, but even if you don't, or if you add `stories.js` files for a n existing component that don't match the above `5` patterns, they will still be added to the Storybook because of this catchall.  

Components that aren't in the `storyTree` will appear either above existing stories (for `storiesOf()` stories) or below all of them (for CSF stories).